cluster/images/hyperkube/master.json

```diff
@@ -23,6 +23,7 @@
               "apiserver",
               "--portal-net=10.0.0.1/24",
               "--address=127.0.0.1",
+              "--read-write-port=8080",
```

Comment:
What is broken? I think the fix is to always use SSL, with a self-signed
cert if needed.

On Wed, Oct 7, 2015 at 2:21 PM, Filip Grzadkowski <notifications@github.com>
wrote:

> In cluster/images/hyperkube/master.json
> <https://github.com/kubernetes/kubernetes/pull/15230#discussion_r41449232>
> :
>
> > @@ -23,6 +23,7 @@
> >                "apiserver",
> >                "--portal-net=10.0.0.1/24",
> >                "--address=127.0.0.1",
> > +              "--read-write-port=8080",
>
> @liggitt <https://github.com/liggitt> AFAIU you're saying that we should
> enforce secured cluster and just break otherwise. I don't think that's the
> way to go. For the same reasons nginx/apache does not enforce TLS, we
> should not either. The problems you're referencing here are because we have
> both ports opened - secure and insecure.
>
> @mikedanese <https://github.com/mikedanese> I'm fine with changing this
> flag to a boolean or even reverse engineering it from other flags - i.e. if
> we expose secure port, then use it, otherwise use insecure.
>
> @lavalamp <https://github.com/lavalamp> I'm not sure you mean by "If we
> can't talk you out of this". Out of what? Currently we're just broken. I'm
> trying to fix this. Also do we even expose port 80? I think this is
> controlled by flags, so I don't think we can assume it's either 443 or 80.
> Am I missing something?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/15230/files#r41449232>.
>